android/src/main/AndroidManifest.xml

```diff
@@ -13,7 +13,8 @@
             android:label="@string/app_name"
             android:roundIcon="@mipmap/ic_launcher_round"
             android:supportsRtl="true"
-            android:theme="@style/Theme.AppCompat.Light.DarkActionBar">
+            android:theme="@style/Theme.AppCompat.Light.DarkActionBar"
+            tools:replace="android:allowBackup,android:supportsRtl">
```

Comment:
why we have this change in this PR? :D don't get it :D

Reply:
In the PR description:

> **Warning**: Small breaking change on Android. Some internal library has `allowBackup:true` and `supportsRtl:true` on their manifest, so we need to add `tools:replace="android:allowBackup,android:supportsRtl"` to Reloaded's `AndroidManifest.xml`


The sample android app also uses `persistence` and was broken by these changes